x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel.test.tsx

```diff
@@ -367,634 +322,634 @@ describe('workspace_panel', () => {
     ] as UserMessage[];
 
     act(() => {
-      mounted.lensStore.dispatch(
+      store.dispatch(
         updateVisualizationState({
           visualizationId: 'testVis',
           newState: {},
         })
       );
     });
-    instance.update();
 
     expect(isSaveable()).toBe(false);
   });
 
-  it('should show proper workspace prompts when auto-apply disabled', async () => {
-    const framePublicAPI = createMockFramePublicAPI();
-    framePublicAPI.datasourceLayers = {
-      first: mockDatasource.publicAPIMock,
-    };
+  describe('enzyme tests', () => {
```

Comment:
I've decided to wrap not rewritten tests with this describe because it's taking me too much time to rewrite all of them.